gulpfile.babel.js

```diff
@@ -207,12 +207,12 @@ gulp.task('build:safari', [
   'build:safari:css',
   'build:safari:js',
   'build:safari:icons',
-  'build:safari:plist'
+  'build:safari:plist',
 ]);
 
 gulp.task('build', [
   'build:webext',
-  'build:safari'
+  'build:safari',
```

Comment:
couldn't we instead change the eslint rule for the last entry ? 
Personally I find it strange that there is a `,` at the end. But thats only my Ruby-brain speaking.

Reply:
Hi @andreasknoepfle, disabling dangling commas was basically the one change that we made over the Airbnb config in our previous, own Bitcrowd config, because it looked so alien to many. At the same time, there's [good reasons](https://github.com/airbnb/javascript#commas--dangling) to actually write the code that way (cleaner diffs and – arguably – easier editing of array/object entries).

There's even languages – [like Go](https://play.golang.org/p/tm7yF0d5hb) – that require this as part of their syntax definition.

If it's not a problem, I'd keep it like this 😃

Reply:
Fine for me 👍  
(But currently not so fine for the CI :P)

Reply:
WAT? But only for the PR job? I'll take a look